client/wfprev-war/src/main/angular/src/app/components/edit-project/project-fiscals/project-fiscals.component.html

```diff
@@ -26,28 +26,30 @@
                 <mat-expansion-panel-header>
                   <div class="header-container">
                     <wfprev-expansion-indicator [isExpanded]="panel.expanded"></wfprev-expansion-indicator>
-                    <span class="project-title">Fiscal Detail</span>
-                    <button class="dropdown-button" mat-icon-button [matMenuTriggerFor]="menu" (click)="$event.stopPropagation()">
-                      <!-- Three-dot menu icon -->
-                      <span>Actions</span>
-                      <img class="action-icon" alt="actions" src="/assets/dropdown-icon-white.svg">
-                    </button>
+                    <span class="project-title">Details</span>
+
+                    <div class="status-actions">
+                      <div class="status-wrapper">
+                        <span>Status:</span>
+                        <wfprev-status-badge
+                          [icon]="getStatusIcon(fiscal.planFiscalStatusCode)"
```

Comment:
planFiscalStatusCode was changed to be a PlanFiscalStatusCodeModel rather than a String in the API, so I don't think this will work without changing to` fiscal.planFiscalStatusCode?.planFiscalStatusCode`

Reply:
done